index.html

```diff
@@ -2497,6 +2502,13 @@ <h3><dfn>Writing content</dfn></h3>
             </li>
           </ol>
         </li>
+        <li>
+          If |context| is `"smart-poster"` and |records| does not contain a
+          <a>URI record</a> for which the protocol is supported by the UA, or
```

Comment:
I think we should reject if more than one URI records exist.

Reply:
fixed